templates/skeleton/app/routes/account/register.tsx

```diff
@@ -1,10 +1,15 @@
 import {
   type ActionFunction,
   type LoaderArgs,
+  type ErrorBoundaryComponent,
```

Comment:
@frandiox Does the v1/v2 script also remove imports, like this one?

I don't think it's a deal breaker if it doesn't, I was just curious.